### PR TITLE
Math exp kpetrov2 patch 1

### PR DESCRIPTION
**Pull request by kpetrov2**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by kpetrov2** [comment]
@SSwiniarski that is the new PR with the requested changes. I made a new branch, because awgraves did commit on the previous branch. I thought he was one of the reviewers.

File: content/c-sharp/concepts/math-functions/terms/exp/exp.md
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/exp/exp.md:20]:
```suggestion
![e formula](https://raw.githubusercontent.com/Codecademy/docs/main/media/e-formula.jpg)
```
The image must be in Markdown.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@kpetrov2 I have one comment,

File: content/c-sharp/concepts/math-functions/terms/exp/exp.md
Comment (kpetrov2) [reply content/c-sharp/concepts/math-functions/terms/exp/exp.md]: Done :)

**Review by SSwiniarski — APPROVED** [review]
@kpetrov2 Looks good.

**Comment by kpetrov2** [comment]
Hi! Should I make any changes?

**Comment by SSwiniarski** [comment]
@kpetrov2 Not yet. The entry needs another review by one of the other maintainers. They may have additional suggestions for you. But you don't need to do anything until then.

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
@kpetrov2 Awesome! Thanks for adding that image:

Screen Shot 2023-02-24 at 1 50 46 PM
Much better 👏 also just checking, did you wish to remain anonymous? If yes, there's nothing else that needs to be done here 🙂

Otherwise, you can do the following:

> If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile.
> 
> To add your GitHub username to your Codecademy profile:
> 
> 1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:
> 
> image
> 
> 2.) Then log out and log back into Codecademy using your GitHub login:
> 
> image

cc: @SSwiniarski

**Comment by kpetrov2** [comment]
Hi! @yangc95 I have followed your instructions, but I'm still ana anonymous contributor, is there another solution?
Best regards!

**Pull request merged**
